modules/create_infrastructure.py

```diff
@@ -45,4 +51,5 @@ def create_infrastructure():
         executor=scheduler_name,
         module_results=module_results,
         container=container_results,
+        params=nfcore_params,
     )
```

Comment:
Needs to include as per #41: 
```suggestion
        cleanup_input=cleanup_input
    )
```